### PR TITLE
[FSK-18]: Configure Storybooks

### PR DESCRIPTION
**Pull request by AmirabbasJ**
this pr adds storybook

### CODE REVIEW

**Comment by AmirabbasJ** [comment]
@erfan-goodarzi @ASafaeirad was this all I had to do?
I mean this ticket had 13 story points, I'm thinking maybe I'm missing something

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<script>
+  window.global = window;
```
Comment (ASafaeirad) [inline .storybook/preview-head.html:2]: Why? :thinking:

**Comment by ASafaeirad** [comment]
It's better to keep the preview file as `.tsx` later on, we need to import some components from src.

File: .storybook/preview-head.html (hunk 1 above)
Comment (AmirabbasJ) [reply .storybook/preview-head.html:2]: https://github.com/storybookjs/storybook/issues/15391

**Pull request merged**